build/bin.NOTICE

```diff
@@ -169,3 +166,71 @@ Apache Commons Lang (lib/org.apache.commons:commons-lang3-<VERSION>.jar)
   
   This product includes software developed at
   The Apache Software Foundation (https://www.apache.org/).
+
+Apache Commons Codec (lib/commons-codec.commons-codec-<VERSION>.jar)
+  Apache Commons Codec
+  Copyright 2002-2017 The Apache Software Foundation
+
+  src/test/org/apache/commons/codec/language/DoubleMetaphoneTest.java
+  contains test data from http://aspell.net/test/orig/batch0.tab.
+  Copyright (C) 2002 Kevin Atkinson (kevina@gnu.org)
+
+  ===============================================================================
+
+  The content of package org.apache.commons.codec.language.bm has been translated
+  from the original php source code available at http://stevemorse.org/phoneticinfo.htm
+  with permission from the original authors.
+  Original source copyright:
+  Copyright (c) 2008 Alexander Beider & Stephen P. Morse.
+
+Apache Commons Logging (lib/commons-logging.commons-logging-<VERSION>.jar)
+  Apache Commons Logging
+  Copyright 2003-2014 The Apache Software Foundation
+
+Apache HttpClient (lib/org.apache.httpcomponents.httpclient-<VERSION>.jar)
+  Apache HttpClient
+  Copyright 1999-2020 The Apache Software Foundation
+
+Apache HttpCore (lib/org.apache.httpcomponents.httpcore-<VERSION>.jar)
+  Apache HttpCore
+  Copyright 2005-2020 The Apache Software Foundation

```

Comment:
We need this Http stuff since it is a dependency of Daffodil, but that's weird. Not sure why http stuff is a depdency of Daffodil. I wonder if that's an optional transitive dependency that we can remove in Daffodil 3.4.0?

Reply:
I removed the http notices because they're not used by daffodil-debugger.  They're in Daffodil's bin.NOTICE only because daffodil-schematron uses Saxon-HE, Saxon-HE uses xml-resolver, and xml-resolver uses these http jars.  You have to build the extension and look at daffodil-debugger's libs as the source of truth before you edit bin.LICENSE or bin.NOTICE.